modules/swagger-codegen/src/main/resources/csharp/compile.mustache

```diff
@@ -17,5 +17,9 @@ if not exist ".\bin" mkdir bin
 
 copy packages\Newtonsoft.Json.8.0.3\lib\{{targetFrameworkNuget}}\Newtonsoft.Json.dll bin\Newtonsoft.Json.dll
 copy packages\RestSharp.105.1.0\lib\{{targetFrameworkNuget}}\RestSharp.dll bin\RestSharp.dll
+copy packages\Fody.1.29.2\Fody.dll bin\Fody.dll
+copy packages\PropertyChanged.Fody.1.51.3\PropertyChanged.Fody.dll bin\PropertyChanged.Fody.dll
+copy packages\PropertyChanged.Fody.1.51.3\Lib\dotnet\PropertyChanged.dll bin\PropertyChanged.dll
+
+%CSCPATH%\csc  /reference:bin\Newtonsoft.Json.dll;bin\RestSharp.dll;bin\Fody.dll;bin\PropertyChanged.Fody.dll;bin\PropertyChanged.dll;System.ComponentModel.DataAnnotations.dll /target:library /out:bin\IO.Swagger.dll /recurse:src\IO.Swagger\*.cs /doc:bin\IO.Swagger.xml
```

Comment:
We can also wrap the dll (related to PropertyChanged) with `{{#generatePropertyChanged}} ... {{/generatePropertyChanged}}`